src/rgw/librgw.cc

```diff
@@ -293,6 +293,25 @@ namespace rgw {
     int http_ret = s->err.http_ret;
 
     req->log_format(s, "http status=%d", http_ret);
+    switch (http_ret / 100) {
```

Comment:
@prashant-kr312 Can you prefix Commit title with "rgw".
>It is required to prefix it with the subsystem or module you are changing.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits